// @sanity/client/dist/stega.js
import { SanityClient, ObservableSanityClient, defineCreateClientExports, middleware } from "./_chunks/nodeMiddleware-CrO2pNEp.js";
import { BasePatch, BaseTransaction, ClientError, ObservablePatch, ObservableTransaction, Patch, ServerError, Transaction, vercelStegaCleanAll } from "./_chunks/nodeMiddleware-CrO2pNEp.js";
import { adapter, environment } from "get-it";
import { encodeIntoResult, stegaEncodeSourceMap } from "./_chunks/stegaEncodeSourceMap-bRxIGJ-b.js";
class SanityStegaClient extends SanityClient {
}
class ObservableSanityStegaClient extends ObservableSanityClient {
}
const exp = defineCreateClientExports(middleware, SanityClient), requester = exp.requester, createClient = exp.createClient;
export {
  BasePatch,
  BaseTransaction,
  ClientError,
  ObservablePatch,
  ObservableSanityClient,
  ObservableSanityStegaClient,
  ObservableTransaction,
  Patch,
  SanityClient,
  SanityStegaClient,
  ServerError,
  Transaction,
  createClient,
  encodeIntoResult,
  requester,
  stegaEncodeSourceMap,
  adapter as unstable__adapter,
  environment as unstable__environment,
  vercelStegaCleanAll
};
